src/Components/RecoProductList/RecoProductList.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState, useEffect } from 'react';
+
+import RecoProduct from 'Components/RecoProductList/RecoProduct/RecoProduct';
+import 'Components/RecoProductList/RecoProductList.scss';
+
+const RecoProductList = () => {
+  const [products, setProducts] = useState();
+  const [direction, setDirection] = useState();
+
+  const getData = async () => {
+    const data = await (await fetch('/data/productData.json')).json();
+    setProducts(data);
+  };
+
+  useEffect(() => getData(), []);
+
+  const handleDirection = e => {
+    const { id } = e.target;
+    setDirection(cur => (cur = id));
```

Comment:
파라미터에 id 를 재할당 해주는 이유가 있나요?
바로 파라미터로 넘겨주어도 됩니다!
```suggestion
    setDirection(id);
```
하지만, 이 방법은 태그에 id를 주는 방법이기 때문에 확장성을 생각하면 다른 방법도 고민해 보는게 좋을 것 같아요!
toggle 스위치를 잘 활용해서 삼항연산자를 사용하는 방법든 여러방법으로 접근을 해 볼 수 있습니다.

Reply:
네 id 대신 className으로 수정해서 값 주었습니다!